src/shared/model/OktaQueryParams.ts

```diff
@@ -0,0 +1,9 @@
+import { StringifiableRecord } from 'query-string';
+
+/**
+ * Okta are query parameters
```

Comment:
wording of comment might need fixing slightly

Reply:
have updated